components/01-globals/typography/label/_label.scss

```diff
@@ -1,10 +1,24 @@
-$label-margin-bottom : 8px !default;
-$label-font-size     : $font-size-small !default;
+$label-margin-top   : 0 !default;
+$label-margin-right : 0 !default;
+$label-margin-bottom: 8px !default;
```

Comment:
maybe it'll be good to use here our margin value divided by 2? What do you think?

Reply:
There was some discussion about it I guess....
![image](https://user-images.githubusercontent.com/12099511/31761685-13f47918-b4b9-11e7-9b5c-e4d37f4dd820.png)
what do you think ?

Reply:
I think that we should keep the vertical spaces unify inside the project. I understand that there can be some specific cases and global variable it's not good everywhere, but in that one, I'd set `$<global-margin> / 2` @Igloczek what do you think, it's already m2 less-bariables-dependencies-tree

Reply:
Good question... I'll suggest keeping value defined directly here and then (as some alpha/beta testing) try to customize some parts of the theme to see if we should change it or not, b/c I'm not sure if the spacing between forms elements should be directly related to i.e. distance between blocks of content.

Reply:
Ok so I leave it this way.